app/config/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "restApiRoot": "/api",
+  "host": "127.0.0.1",
```

Comment:
why not `0.0.0.0` ?

Reply:
I thought about the config as a local dev setting (production could override that). I don't see my local api being reachable over multiple IPs (nevertheless I don't see a problem in changing it to `0.0.0.0`).